src/SelectMenu/SelectMenu.js

```diff
@@ -55,7 +55,7 @@ const SelectMenu = React.forwardRef(({children, initialTab, as, ...rest}, forwar
 
   const onClickOutside = useCallback(
     (event) => {
-      if (!ref.current.contains(event.target)) {
+      if (ref.current && !ref.current.contains(event.target)) {
```

Comment:
huh 🤔 This is super weird - I would have thought that ref would always be defined since we define it up on line 45! Either way can't hurt to have another check!